src/chromium/interoperability-with-cpp/using-cxx-from-chromium.md

```diff
@@ -0,0 +1,16 @@
+## Using cxx from Chromium
+
+The `rust_static_library` target intrinsically knows about cxx interop. Just add
+
+```gn
+cxx_bindings = [ "my_rust_file.rs" ]
```

Comment:
nit: Maybe add a comment saying: "# here we list .rs files that include the `#[cxx::bridge(...)]` declarations / manifests".  (To point out that there is no need to list *all* .rs sources here)

Reply:
Good point, done